segmentation/include/pcl/segmentation/impl/unary_classifier.hpp

```diff
@@ -138,15 +138,14 @@ pcl::UnaryClassifier<PointT>::findClusters (typename pcl::PointCloud<PointT>::Pt
   // find the 'label' field index
   std::vector <pcl::PCLPointField> fields;
   int label_idx = -1;
-  pcl::PointCloud <PointT> point;
   label_idx = pcl::getFieldIndex<PointT> ("label", fields);
```

Comment:
Join this with previous line? (And if possible make it const)

Reply:
Done. A good place for `std::optional`.